coordinator/src/main/scala/filodb.coordinator/DatasetCoordinatorActor.scala

```diff
@@ -168,6 +168,7 @@ private[filodb] class DatasetCoordinatorActor[K](projection: RichProjection[K],
       logger.warn("This should not happen. Not starting flush; $curReprojection; $flushingTable")
       return
     }
```

Comment:
Yeah it confused me, I saw in the code to review, that it ended with:

logger.warn("This should not happen. Not starting flush; $curReprojection; $flushingTable")
return

Odd.